velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -154,8 +156,29 @@ core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
         substraitParser_->parseType(aggFunction.output_type())->type);
     auto aggExpr = std::make_shared<const core::CallTypedExpr>(
         aggVeloxType, std::move(aggParams), funcName);
+
+    const auto& functionSpec = findFunction(aggFunction.function_reference());
+
+    std::vector<TypePtr> rawInputTypes;
+
+    size_t pos = functionSpec.find(":");
+    for (;;) {
+      size_t endPos = functionSpec.find("_", pos + 1);
+      if (endPos == std::string::npos) {
+        std::string typeName = functionSpec.substr(pos + 1);
+        rawInputTypes.push_back(
+            VeloxSubstraitSignature::fromSubstraitSignature(typeName));
+        break;
+      }
+
+      std::string typeName = functionSpec.substr(pos + 1, endPos - pos - 1);
+      rawInputTypes.push_back(
+          VeloxSubstraitSignature::fromSubstraitSignature(typeName));
+      pos = endPos + 1;
+    }
```

Comment:
Locally, we implemented this function to extract types from a function signature.
[SubstraitParser.cpp](https://github.com/oap-project/velox/blob/main/velox/substrait/SubstraitParser.cpp#L295-L320)